scripts/tools/metricExportData.yml

```diff
@@ -0,0 +1,129 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool Experimental Extension for Exporting Metrics Data"
+version: "1.6"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Exporting Metrics Data Experimental Extension Name"
+version: "1.6"
+name: $T_EXPORT_METRICS_DATA_EXP_NAME
+value: '"$XT_experimental_metric_export_data"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Exporting Metrics Data Experimental Extension Version(s)"
+version: "1.6"
+name: $t_export_metric_data_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Maximum count of characters in export data element name"
+version: "1.6"
+name: $T_MAX_METRIC_EXPORT_DATA_ELEMENT_NAME_EXP
+value: "256"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Maximum export data element description string size"
+version: "1.6"
+name: $T_MAX_METRIC_EXPORT_DATA_ELEMENT_DESCRIPTION_EXP
+value: "256"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Metrics calculation descriptor"
+version: "1.6"
+class: $tMetric
+name: $t_metric_calculate_exp_desc_t
+base: $t_base_desc_t
+members:
+    - type: uint32_t
+      name: "rawReportSkipCount"
+      desc: "[in] number of reports to skip during calculation"
```

Comment:
minor: the description should be more explicit saying where are the reports skipped taken from. Beginning, end, middle?